config/webpack.config.base.js

```diff
@@ -25,8 +25,11 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
-        exclude: /node_modules/,
+        test: /\.jsx?$/,
+        include: [
+          path.resolve(__dirname, '../src'),
```

Comment:
I think we must also add our overrides directory here. Probably not in this file however.